### ISSUE TITLE
Yofi backend support ?

### ISSUE THREAD

**Issue by 2Belette** [ayushnix/tessen#36]
Thanks for the app !

Is there any plan to support [yofi](https://github.com/l4l/yofi) as a backend ?

**Comment by ayushnix** [comment]
Sure, shouldn't be a problem. I'll add it in the next release.

**Comment by 2Belette** [comment]
Many thanks very cool !

**Comment by ayushnix** [comment]
yofi support added in version 2.2.0. Let me know if there are any issues.

**Comment by 2Belette** [comment]
Thanks ! I see now yofi as backend support, but I am wondering what I am missing as I see the 'app' yofi menu and not the pass, I imagine a config arguments are missing but I can't find it so far

**Comment by ayushnix** [comment]
@2Belette no problems here, when I run `tessen -d yofi`, I see my list of pass entries.

**Comment by 2Belette** [comment]
thanks ! I tried and got my list of app not pass...
I also deleted my yofi.confi to revert to default and same issue

**Comment by ayushnix** [comment]
@2Belette I copied the new default config over to `$XDG_CONFIG_HOME/tessen/` and I can reproduce this issue as well. Thanks for reporting. I'll try to fix this shortly.

**Comment by ayushnix** [comment]
@2Belette nope, I don't package tessen for Alpine. You'll have to wait or talk to the person who packages it.

**Comment by 2Belette** [comment]
Thanks I left him a message, I let you know when it is ready and I can test it

**Comment by 2Belette** [comment]
@ayushnix after upgrading Alpine to 3.18, I can still use tessen but unable to see the second popup where I can chose login/password/autofill.. anymore.

Don't know how to troubleshoot this as I don't see any verbose/log.

Btw, are you planing to make tessen out from testing ? that is the only package on my entire system on testing would be nice to have it on community so I can remove the edge/testing repo

**Comment by ayushnix** [comment]
@2Belette I'm unable to reproduce this issue. Can you copy the tessen script from `/usr/bin/tessen` to `~/.local/bin/tessen`. Then, edit the script and remove the `set +x` line and on the command line, execute `bash -x ~/.local/bin/tessen` and check the debug output and the error message about why it fails to produce the second popup menu.

> Btw, are you planing to make tessen out from testing ?

As I said before, I haven't packaged tessen for Alpine. You'll have to talk to the Alpine package maintainer for this.

**Comment by 2Belette** [comment]
@ayushnix sorry I missed this one..
So I did what you proposed, I have a huge number of 

```
[2023-07-03][15:18:45][yofi::icon][INFO] failed to load icon at path `"/usr/share/icons/Adwaita/scalable/categories/applications-engineering-symbolic.svg"`: failed to open svg file: "/usr/share/icons/Adwaita/scalable/categories/applications-engineering-symbolic.svg"
```

and then 

```
+ _EXIT_STATUS=0
+ _die
+ [[ -n '' ]]
+ exit 1
+ _clear
+ unset -v _TSN_PASSFILE _TSN_PASSDATA _TSN_USERNAME _TSN_PASSWORD _TSN_URL
+ unset -v _TSN_AUTOTYPE _CHOSEN_KEY
```
I checked and have the adwaita icon theme installed but I don't think this error is the real issue as it is an INFO not an ERROR.

Would you like me to do further tests ?

**Comment by 2Belette** [comment]
I copied the /usr/share/icons/Adwaita/symbolic/categories to /usr/share/icons/Adwaita/scalable/categories and got the error gone..

I am trying to understand why I don't have this scalable folder on Alpine

**Comment by 2Belette** [comment]
I have re-installed adwaita-icon-theme, it seems that it is now including *scalable* directory !
Thanks @ayushnix

**Comment by ayushnix** [comment]
I don't think the adwaita icon theme has anything to do with your issue, that's probably just yofi just expecting an icon theme and not finding one as it expects.

Unfortunately, I'm unable to even launch yofi with tessen right now because yofi keeps panicking with the following error:

```
thread 'main' panicked at 'Format: was already formatted once', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/itertools-0.8.2/src/format.rs:81:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by 2Belette** [comment]
@ayushnix I just did a fresh install of my system, starting from scratch and got the same issue.
Have you been able to solve your issue, or do you want me to do further investigation (same issue where autofill is not doing anything)
